cypress/e2e/page/annotation-pagetests.cy.js

```diff
@@ -0,0 +1,367 @@
+describe("tests on annotation page ui with programmatic state loading and store interaction", () => {
+
+    // List of datasets to use for these tests
+    const datasets = [
+
+        // Good data
+        require("../../fixtures/tests/good-test-config.json")
+
+        // NOTE: Add more dataset json files here via 'require'
+    ];
+
+    beforeEach(() => {
+
+        // Standard app setup
+        cy.appSetup();
+    });
+
+    datasets.forEach((p_dataset) => {
+
+        context("annotation page tests with " + p_dataset.description + " data", () => {
+
+            beforeEach(() => {
+
+                // Setup
+
+                // 1. Open index page
+                // NOTE: Home is visited because some state-related store structures
+                // needed for the categorization page are set up on index page creation
+                cy.visit("/");
+
+                // 2. Load test data
+                cy.loadTestDataIntoStore(p_dataset);
+
+                // 3. Move to the annotation page
+                // NOTE: Routing to the page prevents the Vuex store from being wiped
+                // when a page is 'visited' by Cypress
+                cy.window().its("$nuxt.$router").then(router => {
+
+                    router.push({ path: "/annotation" });
+                });
+            });
+
+            it("simple age annotation", () => {
+
+                // 1. Link 'Subject ID' and 'Age' to data table columns
+                cy.loadAppState("annotation", {
+
+                    categoryColumnPairs: [
+
+                        ["Subject ID", "participant_id"],
+                        ["Age", "age"]
+                    ]
+                });
+
+                // 2. Pause until 'Age' tab (the default annotation tab) components are loaded
+                cy.get("[data-cy='annot-expl-Age']").should("be.visible");
```

Comment:
> That alone might be a reason to look at whether we can allow cy.visit() in the future without resetting the store.

100% This task is specified further in #180 and #179.

> Only use cy.get() to make it clear that we aren't asserting anything yet, just waiting
> Reword the comment to clarify that we're making cypress wait on page load via this workaround, and add a link to > > the cypress docs that explains why this works.

👍

> TODO: flag that we want to revisit this and replace

This should be a standard practice for comments. In fact, I've made an issue about this [here](https://github.com/neurobagel/annotation_tool/issues/183). It would be good to have a formal note somewhere in our code documentation for future contributors (and ourselves) just what these mean and what we intend to do about them – like get them up on github as an issue asap and maybe include the link, for instance. 

>  if we really want to know whether the annotation tab is visible, then assert that separately after this "wait" workaround, so that we know that we want to keep the assertion, but the "wait" workaround can be replaced.

Makes sense.